python/pyspark/sql/dataframe.py

```diff
@@ -4627,11 +4631,22 @@ def dropDuplicates(self, *subset: Union[str, List[str]]) -> "DataFrame":
         +-----+---+------+
         |Alice|  5|    80|
         +-----+---+------+
+
+        Deduplicate values on 'name' and 'height' columns.
+
+        >>> df.dropDuplicates(subset=['name', 'height']).show()
+        +-----+---+------+
+        | name|age|height|
+        +-----+---+------+
+        |Alice|  5|    80|
+        +-----+---+------+
```

Comment:
Hmm this example is exactly the same as the above one. It can be confusing to users whether they should use `dropDuplicates(subset=['name', 'height'])` or directly use `dropDuplicates('name', 'height')`